tempesta_fw/sock_srv.c

```diff
@@ -304,10 +304,17 @@ tfw_srv_conn_release(TfwSrvConn *srv_conn)
 	 * in deferred context after a short pause (in a timer
 	 * callback). The only reason not to start new reconnect
 	 * attempt is removing server from the current configuration.
+	 *
+	 * TFW_CONN_B_ACTIVE bit is checked here to see if the connection has
+	 * already got a server's reference; if so, then server's reference must
+	 * be put. If bit is not set, this means that connection didn't have
+	 * time to get server (due to some error in sock_srv start
```

Comment:
"didn't have time to get server" -> "didn't have time _or was never scheduled_ to reach server"
Using of `get` in this sentence is confusing since talk is about getting/putting reference counters.